cores/nRF5/main.cpp

```diff
@@ -100,6 +100,15 @@ void suspendLoop(void)
   vTaskSuspend(_loopHandle);
 }
 
+void resumeLoop(void)
+{
+  vTaskResume(_loopHandle);
+}
+
+void resumeLoopFromISR(void) {
+  xTaskResumeFromISR(_loopHandle);
```

Comment:
There is no need to for FromISR version, you could detect the ISR context and call the API using `isInISR()` as this 
https://github.com/adafruit/Adafruit_nRF52_Arduino/blob/master/cores/nRF5/utility/SoftwareTimer.cpp#L70

Reply:
Done!